Appraisals

```diff
@@ -1,11 +1,7 @@
-appraise 'rails-5.2' do
```

Comment:
Should we keep Rails 5.2 support for now since it's still supported by the Rails team?

Reply:
Sounds good.